index.js

```diff
@@ -13,6 +13,7 @@ YellowBox.ignoreWarnings([
   'Class GenericShare was not exported. Did you forget to use RCT_EXPORT_MODULE()',
   'Class WhatsAppShare was not exported. Did you forget to use RCT_EXPORT_MODULE()',
   'Class GooglePlusShare was not exported. Did you forget to use RCT_EXPORT_MODULE()',
+  '{"name":"Invariant Violation","framesToPop":1}',
```

Comment:
Are we really meant to ignore this warning? I think it is linked to react-navigation; it is not happy about some stuff. We would really need to refactor it I think.

Reply:
I would love to remove it, anyways, this only helps while useing external debugger so I can just as well remove it